docs/src/pages/components/tabs/NavTabs.js

```diff
@@ -14,9 +14,7 @@ function TabContainer(props) {
   );
 }
 
-TabContainer.propTypes = {
-  children: PropTypes.node.isRequired,
-};
+TabContainer.propTypes = { children: PropTypes.node.isRequired };
```

Comment:
Is this a `typescript` or `typescript-to-proptypes` issue? 

Seems like one or the other is marking this as required if you have `foo?: null | undefined`. It doesn't  matter at runtime but can be confusing if you don't know what `PropTypes.node` accepts.

Reply:
That would be `typescript-to-proptypes`

It doesn't follow definitions with the name `global.JSX.Element`, `React.ReactElement`, or `React.ReactNode`.

I hadn't actually checked the definition for `ReactNode`, I see now that it contains `... | undefined` so I'll update it to treat that as optional. Thanks!

https://github.com/merceyz/typescript-to-proptypes/blob/232634f7782a8119d012ceb69b27e30911ff8052/src/parser.ts#L203-L217